2.6/root/usr/share/container-scripts/mongodb/replset-pre-init.sh

```diff
@@ -0,0 +1,35 @@
+# This file sets some basic mongod replication variables and checks mandatory replication variables
+
+# Add parameters for replication
+export mongod_common_args+=" --oplogSize 64 --replSet ${MONGODB_REPLICA_NAME} "
```

Comment:
Thanks, moved.

It was at the end of file but I moved it during bash sanitization to make it consistent with pre-init.sh. My fault.